teams.md

```diff
@@ -39,11 +39,11 @@ Copy the above code snippet in its entirety and add it to the BOTTOM of this doc
 * [GitHub URL](https://github.com/your-profile/your-project-name)
 ### Project Q&A:
 1. What are you going to build?  
-// your answer here  
+  Making giveaway for creators and users easier and more accessible / propaGATING creators
```

Comment:
Do not delete our example answers. You should answer these questions in your own section.